app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -93,6 +97,7 @@ fun RecentBookmarks(
  * A recent bookmark item.
  *
  * @param bookmark The [RecentBookmark] to display.
+ * @param backgroundColor The background [Color] of the item.
```

Comment:
Could we add the @param for `menuItems` while we are at it?

Reply:
Sure thing